src/algebra/pointwise.lean

```diff
@@ -223,6 +224,14 @@ by { simp only [← inv_preimage, preimage_preimage, inv_inv, preimage_id'] }
 @[simp, to_additive]
 protected lemma univ_inv [group α] : (univ : set α)⁻¹ = univ := preimage_univ
 
+@[simp, to_additive]
+lemma inv_subset_inv [group α] {s t : set α} : s⁻¹ ⊆ t⁻¹ ↔ s ⊆ t :=
```

Comment:
Could you please add `function.surjective.preimage_subset_preimage_iff` so that this proof will becone `(equiv.inv α).surjective.preimage_subset_preimage_iff`?